scripts/size_report

```diff
@@ -48,7 +48,8 @@ parser.add_option("-F", "--rom",
 # to the passed sub-path
 def load_symbols_and_paths(elf_file, path_to_strip = None):
     symbols_paths = {}
-    nm_out = subprocess.check_output(["nm", elf_file, "-S", "-l", "--size-sort", "--radix=d"])
+    bin_nm = os.environ["NM"]
```

Comment:
what if there is no environment variable for nm and objdump? should we fall back to using the executables directly?